modules/gnome/hm.nix

```diff
@@ -0,0 +1,27 @@
+{ pkgs, config, lib, ... }:
+
+with lib;
+
+{
+  options.stylix.targets.gnome.enable =
+    config.lib.stylix.mkEnableTarget "GNOME" true;
+
+  config = mkIf config.stylix.targets.gnome.enable {
```

Comment:
The old GNOME module, based on the User Themes extension, would have supported setting a theme per user. This was changed to a package override to easily add support for GDM, however it should be possible to enable User Themes for the GDM account instead.

Rolling back to User Themes will also avoid conflicts when GNOME is updated.

I can work on implementing this soon.

Reply:
I don't know anything about gnome/dconf, so I'll let you do that ^^